### PR TITLE
Add intro message

### PR DESCRIPTION
**Pull request by KingMaina**
Add a new tests file

### CODE REVIEW

File: last-minute [hunk 1]
```diff
@@ -0,0 +1 @@
+Last minute changes!!
```
Comment (KingMaina) [inline last-minute:1]: Review my last minute changes please @Mercy-Ronoh @ashleySimiyu @KanyivaT

**Review by ashleySimiyu — COMMENTED** [review]
Able to view the changes made

**Comment by KingMaina** [comment]
> Able to view the changes made

Thanks

**Review by ashleySimiyu — APPROVED** [review]
last minute changes reviewed

**Pull request merged**